src/Components/SelfInterview/QuesitonItem.tsx

```diff
@@ -0,0 +1,20 @@
+import { Question } from "../../Apis/types";
+import { AnswersList } from "./AnswersList";
+import "./QuestionItemS.scss";
+
+interface Props {
+    currentQuestion: Question;
+}
+
+export const QuestionItem = ({ currentQuestion }: Props) => {
+    const { answer, question, answers } = currentQuestion;
+
+    return (
+        <div className="QuestionItem">
+            <div>Question: {question}</div>
+            <div>
+                <AnswersList answer={answer} answers={answers} />
+            </div>
```

Comment:
this could have been a little better to pass in the questions and extract them in the list so if we want to add more list functional like correct or select we are able to get them without a small refactor.